primitives/src/constants/ztg.rs

```diff
@@ -9,6 +9,9 @@ pub const COLLATORS: u128 = 10_000_000;
 /// Total ZTG amount for community incentives
 pub const COMMUNITY_INCENTIVES: u128 = 2_000_000;
 
+/// Total ZTG amount for liquidity mining
+pub const LIQUIDITY_MINING: u128 = 10_000_000;
```

Comment:
Is this used anywhere? If so, how is it used?

Reply:
This constant is the total amount of ZTG reserved for the liquidity mining pallet account or in other words, a fund/pot where incentives will be transferred to accounts.

It is defined on the genesis block at https://github.com/zeitgeistpm/zeitgeist/pull/122/files#diff-d0ab461d3f1322691c297bdf74c56e023de548c8555915ed6b60b0a792129341R130-R136

Reply:
Is the pot funded any other ways, i.e. via inflation? This I suppose can be a future addition

Reply:
For now it is a fixed amount that can be changed using a root account but yeah, inflation could be added in a following PR.